src/components/radios/index.md.njk

```diff
@@ -82,21 +82,25 @@ If one or more of your radio options is different from the others, it can help u
 
 {{ example({group: "components", item: "radios", example: "divider", html: true, nunjucks: true, open: false, size: "s"}) }}
 
-### Conditionally revealing content
+### Conditionally revealing a related question
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can ask the user a related question when they select a particular radio option, so they only see the question when it's relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
 
 {{ example({group: "components", item: "radios", example: "conditional-reveal", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-Keep it simple. If you need to add a lot of content, consider showing it on the next page in the process instead.
+Keep it simple. If the related question is complicated or has more than one part, show it on the next page in the process instead.
 
-Do not add conditionally revealing content to inline radios.
+Do not conditionally reveal questions to inline radios, such as ‘yes’ and ‘no’ options placed next to each other.
+
+Conditionally reveal questions only - do not show or hide anything that is not a question.
 
 #### Known issues
 
-Users are not always notified when conditionally revealed content is expanded or collapsed. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html). We plan to fix this issue by the end of June 2021.
+Users are not always notified when a conditionally revealed question is shown or hidden. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html).
+
+However, we found that screen reader users did not have difficulty answering a conditionally revealed question - as long it’s kept simple. Users we tested with did get confused when complicated questions were conditionally revealed to them, particularly questions with more than one part.
```

Comment:
```suggestion
Users, particularly screen readers users, are not always notified when a conditionally revealed question is shown or hidden.

However, our user research found that users did not have difficulty answering conditionally revealed questions - as long they’re kept simple. Users did get confused by complicated questions, particularly questions with more than one part.

Based on our research, [we’re confident that asking conditionally revealed questions, as shown in this component, meets accessibility needs and WCAG 2.1](#).

You should test with users before using conditionally reveals for anything else.

In April 2021, we wrote about how [we explored this issue with web standards and accessibility groups](#).

We support adding the ‘aria-expanded’ attribute to notify the user about a conditionally revealed question, but this is not a part of current standards. We need more user research to prove that this would be helpful to users, and not a distraction.
```